docs/src/main/sphinx/release/release-427.md

```diff
@@ -0,0 +1,110 @@
+# Release 427 (25 Sep 2023)
+
+## General
+
+* Add support for IPv4 addresses in comparisons using the `contains` functions. ({issue}`18497`)
+* Make plans with complex aggregations deterministic. ({issue}`19103`)
+* Improve performance of hashing operations used by `GROUP BY`, `DISTINCT`, and
+  other aggregations. ({issue}`19059`)
+* Fix query failure due to exceeding node memory limits with `INSERT`
+  statements. ({issue}`18771`)
+* Fix incorrectly reporting nanoseconds instead of milliseconds in query plans. ({issue}`19047`)
```

Comment:
This is inaccurate. It's when reporting errors due to timeout during planning. No need to document this.